src/components/PlaidLink/types.ts

```diff
@@ -0,0 +1,25 @@
+import {LinkEventMetadata, LinkSuccessMetadata} from 'react-native-plaid-link-sdk';
+import {PlaidLinkOnEventMetadata, PlaidLinkOnSuccessMetadata, PlaidLinkStableEvent} from 'react-plaid-link';
+
+type PlaidLinkProps = {
+    // Plaid Link SDK public token used to initialize the Plaid SDK
+    token: string;
+
+    // Callback to execute once the user taps continue after successfully entering their account information
+    onSuccess?: (args: {publicToken?: string; metadata: PlaidLinkOnSuccessMetadata | LinkSuccessMetadata}) => void;
+
+    // Callback to execute when there is an error event emitted by the Plaid SDK
+    onError?: (error: ErrorEvent | null) => void;
+
+    // Callback to execute when the user leaves the Plaid widget flow without entering any information
+    onExit?: () => void;
+
+    // Callback to execute whenever a Plaid event occurs
+    onEvent: (eventName: PlaidLinkStableEvent | string, metadata?: PlaidLinkOnEventMetadata | LinkEventMetadata) => void;
```

Comment:
```suggestion
    onEvent: (eventName: string, metadata?: PlaidLinkOnEventMetadata | LinkEventMetadata) => void;
```

PlaidLinkStableEvent is a string

Reply:
Updated